README.md

```diff
@@ -2,15 +2,13 @@
 
 _The official website of the Commonwealth of Massachusetts_
 
-This is the codebase for the Drupal 8 www.mass.gov. The site's theme (mass_theme) is powered by Mayflower, a companion repo available at https://github.com/massgov/mayflower.
-
-[Table of Contents](https://github.com/massgov/DS-Infrastructure/blob/develop/docs/massgov/README.md) for documentation related to this repository such as [Releases](https://github.com/massgov/DS-Infrastructure/blob/develop/docs/massgov/release.md). NOTE: documentation specific to the production site is not publicly available. 
+This is the codebase for the Drupal 8 web site `www.mass.gov`. The site's theme [mass_theme](https://github.com/massgov/openmass/blob/develop/docroot/themes/custom/mass_theme/README.md) is powered by Mayflower, a companion repo available at https://github.com/massgov/mayflower.
```

Comment:
The release link has been removed? I thought we were keeping a link to the release for the release manager.

Reply:
Oops, didn't mean to do that. Will add it back, thanks.